.github/workflows/dev.yml

```diff
@@ -7,7 +7,7 @@ on:
       - synchronize
       - reopened
     branches:
-      - main
+      - 'release/*'
```

Comment:
This maybe invalid in this pr. @everpcpc  please review it.

Reply:
this PR is being merged into branch [release/v1.2.636-rc3](https://github.com/databendlabs/databend/tree/release/v1.2.636-rc3), instead of the main branch